life/modernized/18-chapter-18.adoc

```diff
@@ -405,7 +411,8 @@ It continued about five hours; he affirmed in opposition to us,
 that there was no Christian church, ministry, or commission apostolical now in the world;
 but the Lord assisted us with his wisdom and strength, to confound his attempts.
 
-Here ends the narrative.
+[.alt.centered]
+==== Here ends the narrative.
```

Comment:
This may not be the best way to label these because it puts this on more on top of the paragraph below it, rather than finishing off the paragraph above it. Again, the [.offset] looked much better after I went back in a tried them.